sources/models/shared-defaults/kexec-load-disabled.toml

```diff
@@ -0,0 +1,3 @@
+# Turn off kexec, even if it's built in.
+[settings.kernel.sysctl]
+"kernel.kexec_load_disabled" = "1"
```

Comment:
These should go away with the change in `kexec_load_disabled` handling that I suggested.

Reply:
^ I like that plan, but if it doesn't work for some reason, I think we'll want a migration to remove this sysctl setting on downgrade.